nodejs-async/nodejs-async.spec

```diff
@@ -0,0 +1,49 @@
+%global npm_name async
+%global enable_tests 1
+
+%{?nodejs_find_provides_and_requires}
+
+Name: nodejs-%{npm_name}
+Version: 0.2.10
+Release: 1%{?dist}
+Summary: Higher-order functions and common patterns for asynchronous code
+License: MIT
+URL: https://github.com/caolan/async/issues
+Source0: http://registry.npmjs.org/%{npm_name}/-/%{npm_name}-%{version}.tgz
+BuildRequires: nodejs-packaging
+BuildArch:  noarch
+
+%if 0%{?fedora} >= 19
+ExclusiveArch: %{nodejs_arches} noarch
+%else
+ExclusiveArch: %{ix86} x86_64 %{arm} noarch
+%endif
+
+%{?nodejs_find_provides_and_requires}
+
+%description
+%{summary}
+
+%prep
+%setup -q -n package
+
+%build
+%nodejs_symlink_deps --build
+
+%install
+mkdir -p %{buildroot}%{nodejs_sitelib}/%{npm_name}
+cp -pfr LICENSE README.md component.json lib package.json %{buildroot}%{nodejs_sitelib}/%{npm_name}
+%nodejs_symlink_deps
+
+%if 0%{?enable_tests}
+%check
+%{nodejs_symlink_deps} --check
+%endif
+
+%files
+%{nodejs_sitelib}/%{npm_name}
+
+%doc LICENSE
```

Comment:
```
%files
%doc LICENSE
....

%files doc
%doc README.md
```

preferred, one files section for docs and another for regular files, license and binaries